lib/mix/tasks/dogma.ex

```diff
@@ -11,8 +11,11 @@ defmodule Mix.Tasks.Dogma do
     {dir, formatter} = argv |> parse_args
     config = Config.build
 
+    {:ok, events} = GenEvent.start_link([])
```

Comment:
If one of the reporters crashes, but all of the rules pass then it could makes sense in a CI environment. If none of the rules fail then the build should still be green. But that's a pretty narrow use case; I think for the most part people are going to want to see reports.